README.md

```diff
@@ -2,7 +2,7 @@
 
 OpenShift Helm Charts is a repository hosting [Helm Charts](https://github.com/helm/helm) available by default with [OpenShift](https://www.openshift.com/). It contains popular technologies, tools and services. Helm Charts on this repository can be provided by the community, by partners or Red Hat. 
 
-Charts go through an automated RedHat OpenShift certification workflow, which guarantees security compliance as well as best integration and experience with the platform.
+Charts go through an automated Red Hat OpenShift certification workflow, which guarantees security compliance as well as best integration and experience with the platform.
```

Comment:
Looks good can you also send a fix to the gh-pages branch? That is what we see publicly. Thanks